src/storage/src/hummock/store/hummock_storage.rs

```diff
@@ -488,11 +489,16 @@ impl StateStoreRead for HummockStorage {
 
     async fn iter_log(
         &self,
-        _epoch_range: (u64, u64),
-        _key_range: TableKeyRange,
-        _options: ReadLogOptions,
+        epoch_range: (u64, u64),
+        key_range: TableKeyRange,
```

Comment:
I should have asked in the previous PR. Why do we need `key_range` in iter_log?

Reply:
Currently it will be used to specified the vnodes it wants to read.